### ISSUE TITLE
DioError [DioErrorType.DEFAULT]: FileSystemException: Cannot retrieve length of file, path = 'file:///var/mobile/Containers/Data

### ISSUE THREAD

**Issue by view-man** [flutterchina/dio#567]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

DioError [DioErrorType.DEFAULT]: FileSystemException: Cannot retrieve length of file, path = 'file:///var/mobile/Containers/Data

**Comment by view-man** [comment]
IOS 录音之后上传音频有问题

**Comment by ShshyDevooo** [comment]
请问问题解决了吗，我也遇到同样的问题

**Comment by view-man** [comment]
解决了



---Original---

**Comment by ShshyDevooo** [comment]
> 解决了
> […](#)
> ---Original--- From: "DevSSY"<notifications@github.com&gt; Date: Fri, Nov 22, 2019 09:36 AM To: "flutterchina/dio"<dio@noreply.github.com&gt;; Cc: "孙益飞"<645139778@qq.com&gt;;"Author"<author@noreply.github.com&gt;; Subject: Re: [flutterchina/dio] DioError [DioErrorType.DEFAULT]: FileSystemException: Cannot retrieve length of file, path = 'file:///var/mobile/Containers/Data (#567) 请问问题解决了吗，我也遇到同样的问题 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

请问是如何解决的啊？

**Comment by view-man** [comment]
> > 解决了
> > […](#)
> > ---Original--- From: "DevSSY"<[notifications@github.com](mailto:notifications@github.com)> Date: Fri, Nov 22, 2019 09:36 AM To: "flutterchina/dio"<[dio@noreply.github.com](mailto:dio@noreply.github.com)>; Cc: "孙益飞"<[645139778@qq.com](mailto:645139778@qq.com)>;"Author"<[author@noreply.github.com](mailto:author@noreply.github.com)>; Subject: Re: [flutterchina/dio] DioError [DioErrorType.DEFAULT]: FileSystemException: Cannot retrieve length of file, path = 'file:///var/mobile/Containers/Data (#567) 请问问题解决了吗，我也遇到同样的问题 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> 请问是如何解决的啊？

file:///var/mobile/Containers/Data
iOS的路径问题，应该不是dio的事情，应该是录音插件的问题，你把file://截掉就好了
